_data/unternehmen.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "title": "Data Science as a Service",
+    "caption": "Wir bringen Big Data Know-how vom CERN in die Wirtschaft und unterstützen Unternehmen und Organisationen mit Datenanalyse, Machine Learning und Simulationen.",
+    "url": "/img/logos/point8.svg",
```

Comment:
maybe call it `homepage` and `logo` to make clear what the urls should point to?